src/core/utilities/cellmap.hpp

```diff
@@ -136,18 +136,18 @@ class CellMap
 
 
     // export from 'from' into 'dest' items indexed in the map found withing 'box'
-    template<typename Array>
-    void export_to(box_t const& box, Array const& from, Array& dest) const;
+    template<typename Src, typename Dst>
+    void export_to(box_t const& box, Src const& from, Dst& dest) const;
 
 
     // same as previous but applies a transformation to the items before exporting to 'dest'
-    template<typename Array, typename Transformation>
-    void export_to(box_t const& box, Array const& from, Array& dest, Transformation&& Fn) const;
+    template<typename Src, typename Dst, typename Transformation>
+    void export_to(box_t const& box, Src const& from, Dst& dest, Transformation&& Fn) const;
 
 
     // export items satisfying Predicate in 'from' into 'dest'
-    template<typename Array, typename Predicate>
-    void export_to(Array const& from, Array& dest, Predicate&& pred) const;
+    template<typename Src, typename Dst, typename Predicate>
+    void export_to_v2(Src const& from, Dst& dest, Predicate&& pred) const;
```

Comment:
without renaming this function or possibly some SFINAE magic, this function was conflicting with the first one